src/prepare.js

```diff
@@ -179,6 +185,111 @@ function getConfigOverride(confStr) {
 	return config;
 }
 
+/**
+ * @param {ReturnType<getConfigOverride>} conf
+ * @param {ReturnType<typeof getBasicBuildOptions>["toolchain"]} toolchain
+ */
+async function extendConfig(conf, toolchain) {
+	if (!conf) return;
+	if (toolchain === "respec" && !conf.specStatus) return;
+	if (toolchain === "bikeshed" && !conf.status) return;
```

Comment:
note that specStatus / status could be inferred by default using the rules suggested in #13 (but obivously doesn't need to be part of that PR)

Reply:
Yes. I've that working locally now :smile: